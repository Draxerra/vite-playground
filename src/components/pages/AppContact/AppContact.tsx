import React, { FC } from "react";

const AppContact: FC = () => (
  <section>
    <h2 className="fs-600">Contact Page</h2>
  </section>
);

export default AppContact;
