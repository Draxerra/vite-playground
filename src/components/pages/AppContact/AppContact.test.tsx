import React from "react";
import AppContact from "./AppContact";
import { render } from "@testing-library/react";

describe("App Contact", () => {
  it("should match snapshot", () => {
    const contact = render(<AppContact />).asFragment();
    expect(contact).toMatchSnapshot();
  });
});
