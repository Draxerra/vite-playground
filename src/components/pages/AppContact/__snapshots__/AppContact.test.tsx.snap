// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Contact should match snapshot 1`] = `
<DocumentFragment>
  <section>
    <h2
      class="fs-600"
    >
      Contact Page
    </h2>
  </section>
</DocumentFragment>
`;
